components/Card.js

```diff
@@ -95,11 +106,37 @@ export default function Card({ date, id, weights, handleDelete, heights, feastTi
           </AttributeList>
         </>
       )}
+      {diaperColors.length !== 0 && (
+        <>
+          <AttributeText>Diaper Color</AttributeText>
+          <AttributeColorList>
+            {diaperColors.map((diaperColor) => (
+              <AttributeColorListItem key={diaperColor._id}>
+                {pathname === '/' && (
+                  <DeleteSingleColor
+                    inputColor={diaperColor.value}
+                    aria-label="delete single data point"
```

Comment:
Is this aria label really meaningful? If I understand the use correctly, for me this should say "Delete single color".